Makefile

```diff
@@ -5,7 +5,7 @@ PREFIX = /usr/local
 SHAREPREFIX = ${PREFIX}/share/lighthouse
 DOLLAR = $$
 
-CC=gcc
+# CC=gcc
```

Comment:
not sure if you meant to include this, perhaps make this `CC?=gcc`

Reply:
Fixed.